MetaMask/metamask-extension
# feat: add Mint Blockchain

## **Description**

Add Mint Blockchain Mainnet network icon support.

## **Related issues**

Fixes: N/A

## **Manual testing steps**

1. Go to [Mint](https://www.mintchain.io)
2. Click the Metamask icon to add network
3. Open Metamask extension to check the network and token icons

## **Screenshots/Recordings**

### **Before**
<img width="371" alt="image" src="https://github.com/user-attachments/assets/acab6b3b-6636-4668-b487-acdd8050685f">

### **After**
![mint-metamask-white](https://github.com/user-attachments/assets/39454de4-cbfc-4a43-aafa-4746f69de191)

![mint-metamask-black](https://github.com/user-attachments/assets/53adce6c-dbee-4d05-b7bb-ed3984e38be0)

## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
